src/components/Shared/Image/DeleteIcon.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { DELETE_ICON_ALT_TEXT, ICON_DELETE } from "utils/constant/images";
+
+const DeleteIcon = ({ action }) => {
```

Comment:
what does the prop `action` represent?

// see https://react.dev/learn/responding-to-events#naming-event-handler-props